src/pages/home.tsx

```diff
@@ -31,7 +32,10 @@ export const HomePage: React.FunctionComponent = () => {
 
   return (
     <div className="bg-gray-100">
-      <DropZone setRawDocument={(doc) => setRawDocument(doc)} />
+      <div className="flex">
+        <DemoDocument setRawDocument={(document) => setRawDocument(document)} />
+        <DropZone setRawDocument={(document) => setRawDocument(document)} />
```

Comment:
I never noticed that ... setRawDocument is coupling to what the user intend to do, it should be  named sth like => onFileLoaded, onDocumentLoaded, onDocumentDropped,